person.rb

```diff
@@ -2,7 +2,7 @@
 
 # Person class
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rental
```

Comment:
- [ ] Kindly make the `rental` attribute have a getter and setter methods. 👍

Reply:
`rental` in person is a `list` or `array` and is updated via `rental` object/class. Do we still need a setter for this?